.github/workflows/node-ci-push.yml

```diff
@@ -68,3 +68,100 @@ jobs:
       with:
         token: ${{ secrets.CODECOV_TOKEN }}
         files: ./coverage/coverage-final.json
+
```

Comment:
These changes are based on chatting with Lilli on how to work with [Intermediate Integration Branches](https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/2918416504/Understand+our+Branching+Strategy#Example-2---Intermediate-Integration-Branch). These changes will make it so `main` will include these checks and we can include commits from `main` into feature branches. Please let me know if there's another way to address this so that we're able to merge the latest changes from main into feature branches!